cmd/checkconfig/main.go

```diff
@@ -977,7 +977,7 @@ func validateNeedsOkToTestLabel(cfg *config.Config) error {
 							"the tide query at position %d"+
 								"forbids the %q label and requires the %q label, "+
 								"which is not recommended; "+
-								"see https://docs.prow.k8s.io/docs/components/core/tide/maintainers/#best-practices "+
+								"see https://github.com/kubernetes-sigs/prow/blob/main/site/content/en/docs/components/core/tide/maintainers.md#best-practices "+
```

Comment:
https://docs.prow.k8s.io/docs/components/core/tide/maintainers/#best-practices works, IMO no need to migrate it

Reply:
I reverted to https://docs.prow.k8s.io/docs/components/core/tide/maintainers/#best-practices in 29341e9, but the reverting caused unit test error..
I might drop the commit 29341e9 and push again. Does this seem ok? :)

Reply:
I have updated `checkconfig` testcode in d3b59c2 to suppress error, because
IMO the testcode should be updated not to return error which occurs when the regexp `[^/]k8s[^/]` matches the error string,
where the error string in this case is:

```
"the following enable strict branchprotection context requirements even though Tide handles merges: [org: kubernetes]. See #5: https://docs.prow.k8s.io/docs/components/core/tide/maintainers/#best-practices Also note that this validation is imperfect, see the check-config code for details"
```

Please take a look.

Reply:
Documentation links should land at the Prow docs site (https://docs.prow.k8s.io), not the github source .md files, since the docs site provide a far superior user experience.

If we need to improve tests in some way to allow this, we should take that course.